modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
+= Binding infrastructure node workloads using taints and tolerations
+
+If you have an infra node that has the `infra` and `worker` roles assigned, you
+must configure the node so that user workloads are not assigned to it.
+
+.Prerequisites
+
+* Configure additional MachineSets in your {product-title} cluster.
+
+.Procedure
+
+. Use the following command to add a taint to the infra node to prevent
+scheduling user workloads on it:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> <key>:<effect>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc adm taint nodes node1 node-role.kubernetes.io/infra:NoSchedule
+----
++
+This example places a taint on `node1` that has key
+`node-role.kubernetes.io/infra` and taint effect `NoSchedule`. Nodes with the
+`NoSchedule` effect schedule only Pods that match the taint, but allow existing
+Pods to remain scheduled on the node.
+
+. Add tolerations for the Pod configurations you want to schedule on the infra
```

Comment:
In addition to the toleration the pod must be updated so it's scheduled on infra, else it will be scheduled to any worker.  This might include infra but isn't guaranteed.    Examples can be found here for `nodeSelector`
https://docs.openshift.com/container-platform/4.4/machine_management/creating-infrastructure-machinesets.html#moving-resources-to-infrastructure-machinesets

There are a lot of options for how to do this but having an explicit example here will help ensure it's not a missed step.  For more advanced configurations reference https://docs.openshift.com/container-platform/4.4/nodes/scheduling/nodes-scheduler-about.html

Reply:
@jewzaam This new module is situated directly before [Moving resources to infrastructure MachineSets](https://docs.openshift.com/container-platform/4.4/machine_management/creating-infrastructure-machinesets.html#moving-resources-to-infrastructure-machinesets). Instead of reinventing the wheel, I've added some segue text to lead into those sections for scheduling infra resources with `nodeSelector`. Let me know if you have any objections to that structure.

Reply:
Recommend adding a list item here to reflect the need for the scheduling component else it implies tolerations are sufficient.